torch/utils/_config_module.py

```diff
@@ -165,10 +196,19 @@ class _ConfigEntry:
     user_override: Any = _UNSET_SENTINEL
     # The justknob to check for this config
     justknob: Optional[str] = None
+    # environment variables are read at install time
```

Comment:
BTW, this is a problem for future us, but handling TRITON_CACHE_DIR is going to be irritating because it is NOT read at install time and we regularly change the environ to make changes. It's better for the code we write to be patching envvar, but it may be more BC to continually read out of envvar.